java/test.sh

```diff
@@ -27,6 +27,10 @@ echo "Running tests under cluster mode."
 # bazel test //java:all_tests --action_env=ENABLE_MULTI_LANGUAGE_TESTS=1 --test_output="errors" || cluster_exit_code=$?
 ENABLE_MULTI_LANGUAGE_TESTS=1 run_testng java -cp $ROOT_DIR/../bazel-bin/java/all_tests_deploy.jar org.testng.TestNG -d /tmp/ray_java_test_output $ROOT_DIR/testng.xml
 
+echo "Running tests under cluster mode with 5 workers per Java worker process."
```

Comment:
I think we can change the default value of `num_workers_per_process_java` to be more than 1, (e.g. 10). This should make sense for most Java applications.
And then we don't need to add this test here.

Reply:
Done.